### PR TITLE
Add count flag for search command

### PR DESCRIPTION
**Pull request by AlexeiVainshtein**

### CODE REVIEW

File: artifactory/cli.go [hunk 1]
```diff
@@ -952,6 +952,10 @@ func getSearchFlags() []cli.Flag {
 			Name:  "build",
 			Usage: "[Optional] If specified, only artifacts of the specified build are matched. The property format is build-name/build-number. If you do not specify the build number, the artifacts are filtered by the latest build number.` `",
 		},
+		cli.BoolFlag{
+			Name:  "count",
+			Usage: "[Optional] Set to true if you do not wish to print the entire JSon result but only the total of found artifacts.` `",
```
Comment (eyalbe4) [inline artifactory/cli.go]: I suggest changing the description to:
*Set to true to display only the total of files or folders found.*

File: artifactory/cli.go [hunk 2]
```diff
@@ -1771,7 +1775,9 @@ func searchCmd(c *cli.Context) {
 	cliutils.ExitOnErr(err)
 	result, err := json.Marshal(searchCmd.SearchResult())
 	cliutils.FailNoOp(err, len(searchCmd.SearchResult()), 0, isFailNoOp(c))
-	log.Output(string(clientutils.IndentJson(result)))
+	if !c.Bool("count") {
```
Comment (eyalbe4) [inline artifactory/cli.go]: Let's changes this, so that the files / folders count is displayed.

File: artifactory/cli.go (hunk 1 above)
Comment (AlexeiVainshtein) [reply artifactory/cli.go]: Done

File: artifactory/cli.go (hunk 2 above)
Comment (AlexeiVainshtein) [reply artifactory/cli.go]: Done

**Pull request merged**